WordPress/gutenberg
# Query Loop: Remove CreateNewPostLink component from inspector controls

Fixes: #67177 

## What?
This PR addresses an enhancement mentioned in the Issue Description which enforces removal of the Add New Post link within the Query Loop Block to clear user ambiguity.

## How?
The PR just removes the Component being used in the index file and deletes the component definition.

## Testing Instructions
1. Create a new Query Loop Block.
2. Notice there's no Add New Post link on the Query Loop's Block info.

## Screenshot

![Screenshot 2024-11-21 at 11 47 58 AM](https://github.com/user-attachments/assets/e325dcae-c154-4e98-bb58-0df63601b439)
